pkg/kubelet/server/stats/fs_resource_analyzer.go

```diff
@@ -89,7 +89,9 @@ func (s *fsResourceAnalyzer) updateCachedPodVolumeStats() {
 		podUid := pod.GetUID()
 		stats, found := s.getPodVolumeStats(pod)
 		if !found {
-			glog.Warningf("Could not locate volumes for pod %s", format.Pod(pod))
+			if pod.Status.Phase == api.PodRunning {
```

Comment:
Does this mean we will not be collecting stats until all the containers are running?